src/ripple/app/paths/cursor/RippleLiquidity.cpp

```diff
@@ -74,6 +74,9 @@ void rippleLiquidity (
     assert (saPrvReq.getCurrency () == saPrvAct.getCurrency ());
     assert (saPrvReq.getIssuer () == saPrvAct.getIssuer ());
 
+    STAmountCalcSwitchovers amountCalcSwitchovers (
+        rippleCalc.view.info ().parentCloseTime);
```

Comment:
Consider moving where `amountCalcSwitchovers` is constructed to just before the first use.